content/chat/rooms/index.textile

```diff
@@ -0,0 +1,121 @@
+---
+title: Rooms
+meta_description: "Use rooms to organize your users and chat messages."
+product: chat
+languages:
+  - javascript
+---
+
+Rooms are used to organize and logically separate your users and chat messages into 'rooms'. They are the entry object into using chat and provide access to all other chat features, such as messages, online status and typing indicators. A room can represent a 1:1 chat between an agent and a customer, a private message between two users in a chat application, a large group conversation, or the chat section of a livestream with thousands of users.
+
+You also control which features are enabled in a room when you create one. For instance, you might only want to utilize typing indicators in direct messages between users.
+
+h2(#create). Create or retrieve a room
+
+Users send messages to a room and subscribe to the room in order to receive messages. Other features, such as indicating which users are online, or which users are typing are configured as part of a room's options.
+
+A @room@ is created, or an existing one is retrieved from the @rooms@ collection using the "@rooms.get()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Rooms.html#get method:
```

Comment:
Link is now broken. Must be:

https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Rooms.html#get.get-1

Same for all others I think?

**Slack threads to see before doing changes**

Also question about versioning on slack: https://ably-real-time.slack.com/archives/C02NY1VT3LY/p1720616674027369

And question about whether the new links is what we really want: https://ably-real-time.slack.com/archives/C02NY1VT3LY/p1720616533297169